### DISCUSSION TITLE
How can I use part of megadepth for training?

### DISCUSSION THREAD

**Discussion by HUSTNO1WXY** [zju3dv/LoFTR#73] [Q&A] [answered]
Can I use smaller part of dataset for training?

**Comment by zehongs** [comment, marked as answer]
You can modify the contents of https://github.com/zju3dv/LoFTR/blob/94e98b695be18acb43d5d3250f52226a8e36f839/configs/data/megadepth_trainval_840.py#L8

**Reply by HUSTNO1WXY** [reply]
> You can modify the contents of
> 
> https://github.com/zju3dv/LoFTR/blob/94e98b695be18acb43d5d3250f52226a8e36f839/configs/data/megadepth_trainval_840.py#L8

I don't download the whole megadepth and modify this train_list.txt, can I ask the meaning of numbers in this txt?

**Comment by zehongs** [comment]
`sequence-id` + `min-overlap-score` + `max-overlap-score`
